source/Engine.cpp

```diff
@@ -586,15 +586,24 @@ void Engine::Step(bool isActive)
 		}
 	}
 
-	// Draw a highlight to distinguish the flagship from other ships.
+	outlines.clear();
+	const Color &cloakColor = *GameData::Colors().Get("cloak highlight");
+	if(Preferences::Has("Cloaked ship outlines"))
+		for(const auto &ship : ships)
+		{
+			if(!ship->IsYours() || ship->GetSystem() != player.GetSystem() || ship->Cloaking() == 0.)
```

Comment:
playerinfo has a list of all player ships, wouldnt it make more sense to just add a getter for that list?

I imagine that would have use cases in other parts of the code as well

Reply:
I don't see how this is relevant to this PR?

Reply:
Nevermind I missed the!

Reply:
Wait no, it is relevant. Because you only want to loop through the player ships

Reply:
`PlayerInfo::Ships` is already a public method. And, it does look like this should just be iterating over that instead of hte list of all ships given that it skips any ships that isn't the player's.